model/task_config.go

```diff
@@ -142,7 +142,16 @@ func MakeConfigFromTask(t *task.Task) (*TaskConfig, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "error finding host")
 	}
-	e, err := PopulateExpansions(t, h)
+
+	settings, err := evergreen.GetConfig()
```

Comment:
This function is only called from host.create. Why are you modifying it in this PR?